dynawo/sources/Modeler/Common/DYNModelMulti.h

```diff
@@ -462,6 +462,13 @@ class ModelMulti : public Model, private boost::noncopyable {
    */
   void setCurrentZ(const std::vector<double>& z);
 
+  /**
+   * @brief set local initialization solver parameters
+   *
+   * @param localInitParameters local initialization solver parameters
+   */
+  void setLocalInitParameters(boost::shared_ptr<parameters::ParametersSet> localInitParameters);
```

Comment:
Why using a setter here instead of passing it into the init method?

Reply:
This setter enables us to set localInitParameters_ in the Simulation::initFromData(const shared_ptr<DataInterface>& data, const shared_ptr<DynamicData>& dyd) method which is before the ModelMulti::init(const double t0) method.

Reply:
why is that required? Cannot we call ModelMulti::init(t0, localInitParam)?

Reply:
I can put this code : 

```
    const std::string initParFile = createAbsolutePath(jobEntry_->getLocalInitEntry()->getParFile(), context_->getInputDirectory());
    const std::string parId = jobEntry_->getLocalInitEntry()->getParId();
    parameters::XmlImporter parametersImporter;
    boost::shared_ptr<ParametersSetCollection> localInitSetCollection = parametersImporter.importFromFile(initParFile);
    boost::shared_ptr<ParametersSet> localInitParameters = localInitSetCollection->getParametersSet(parId);
```

just before the call of ModelMulti::init(const double t0) to put localInitParameters as an argument of the method but it will be less neat. It will be more scattered and less divided into functions.

Reply:
We could just store localInitParameters as an attribute of DYNSImulation as it is done for others things?

Reply:
We can but there's no other ParametersSet attributes in Simulation class.

Do you think the current implementation is less efficient than what it should could be ?